src/main/scala/junctions/crossing.scala

```diff
@@ -91,3 +90,46 @@ object ClockToSignal {
     c2s.io.signal_out
   }
 }
+
+object LevelCrossing {
```

Comment:
What is this for? "LevelCrossing" is a vague name to me... do you mean "Synchronizer for level signals" or do you mean "Power Domain Level Crossing"?

In general, for "library-esque" things like this, please write some good description of what this is and how to use it.